nhaystack-rt/src/nhaystack/driver/history/BNHaystackHistoryExport.java

```diff
@@ -205,9 +205,9 @@ private HRef locatePoint(HClient client)
   {
     BHistoryId id = getHistoryId();
     StringBuilder pointFilter = new StringBuilder();
-    pointFilter.append("point and axStation==");
+    pointFilter.append("point and n4Station==");
     pointFilter.append(HStr.make(Sys.getStation().getStationName()).toZinc());
-    pointFilter.append(" and axHistoryId==");
+    pointFilter.append(" and n4HistoryId==");
```

Comment:
Same as previous comment about trying to achieve backwards compatibility with existing stations.